spec/features/user_searches_places_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature 'A user searches places' do
+  scenario 'and views matching places' do
+    irrelevant_places = create_list(:place, 5, :active, name: 'Playmoweb')
+    relevant_places = create_list(:place, 2, :active, name: 'Craftsmen')
```

Comment:
Useless assignment to variable - `relevant_places`. Did you mean `irrelevant_places`?